Python_Scripts/loadModels.py

```diff
@@ -3,9 +3,15 @@
 import sys
 import os
 import importlib
+import pandas as pd
+import numpy as np
 
-from C import DIR_MODELS_AMICI
+import libsbml
+from C import DIR_MODELS_REGROUPED, DIR_MODELS_AMICI, DIR_MODELS
 
+# import the model_summary table to add information about the model
+model_info = pd.read_csv(os.path.join(DIR_MODELS, 'model_summary.tsv'),
```

Comment:
I rather don't like global variables in a file declaring constants (i.e. not script file). define in caller and pass as arg?

Reply:
well, agreed... is defined in caller anyway, will pass it